### PR TITLE
Update CI and related files to match module template

### PR DESCRIPTION
**Pull request by Mrtenz**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+* See: #67890
+-->
+
+## Examples
+
+<!--
+Are there any examples of this change being used in another repository?
+
+When considering changes to the MetaMask module template, it's strongly preferred that the change be experimented with in another repository first. This gives reviewers a better sense of how the change works, making it less likely the change will need to be reverted or adjusted later.
+-->
```
Comment (mcmire) [inline .github/pull_request_template.md]: This belongs to the module template.

``` suggestion
```

**Review by mcmire — COMMENTED** [review]
One suggestion, but otherwise looks good.

File: .github/pull_request_template.md (hunk 1 above)
Comment (Mrtenz) [reply .github/pull_request_template.md]: Good catch.

**Comment by Mrtenz** [comment]
`auto-changelog` is missing, causing CI to fail. Will address that in a future PR.

**Review by mcmire — APPROVED** [review]
Looks good!

**Pull request merged**